BenchPress/Helpers/Azure/BenchPress.Azure.psd1

```diff
@@ -23,6 +23,7 @@
   )
   FunctionsToExport = @(
     "Invoke-AzCli",
+    "Confirm-BicepFile",
```

Comment:
Did you consider Validate-BicepFile as the name of this feature?

Reply:
Validate is not a valid Verb for PowerShell and will throw a PS Script Analyzer error.